Emulator/Sound/TCoreAudioSoundManager.cpp

```diff
@@ -42,7 +42,7 @@
 // Einstein.
 #include "Emulator/Log/TLog.h"
 
-#define MIN(a,b) ((a) < (b) ? (a) : (b))
+#define EINSTEIN_MIN(a,b) ((a) < (b) ? (a) : (b))
```

Comment:
If it's just used once, we could move it down/inline it.

Reply:
I changed it to std::min(). Easy. One other project in C++ rejects all std:: and template features. Stressful!